prisma/schema.prisma

```diff
@@ -0,0 +1,28 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
```

Comment:
Change provider also to env

Reply:
couldn't change... it says that provider must be a string literal